content/projects/jcasc/index.adoc

```diff
@@ -39,7 +39,7 @@ Fully working Jenkins master with:
 === Getting started
 
 * link:https://github.com/jenkinsci/configuration-as-code-plugin/blob/master/README.md[*Documentation and getting started*]
-* You can ask questions about it on the regular Jenkins Developer mailing list (just prefix with [JCasC]).
+* You can ask questions about it on the regular Jenkins Users mailing list (just prefix with [JCasC]).
```

Comment:
```suggestion
* Questions about the **use** of configuration as code should be asked on the Jenkins Users mailing list (prefix the subject line with [JCasC]).
* Questions related to the **development** of the plugin should be asked on the Jenkins Developers mailing list  (just prefix with [JCasC]).
```

Reply:
+1

Reply:
@daniel-beck have you considered this suggestion and decided against it?  I'm fine with that choice, just need to know your choice before the merge.

Reply:
No opinion, either way works for me. If this makes the change more palatable to others, sure.

Reply:
@daniel-beck then you'll need to be the one to commit the suggestion.  I don't have permission to commit to your repository.